modules/product/src/Plugin/Field/FieldWidget/ProductVariationAttributesWidget.php

```diff
@@ -239,7 +239,7 @@ protected function getAttributeInfo(ProductVariationInterface $selected_variatio
       $field = $field_definitions[$field_name];
       $attributes[$field_name] = [
         'field_name' => $field_name,
-        'title' => $field->getLabel(),
+        'title' => $this->t($field->getLabel()),
```

Comment:
There's no need to wrap this in translatable markup, as the field is config and config is already translatable